packages/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -165,15 +161,10 @@ import dayjs, { Dayjs } from 'dayjs'
 import DateTable from './basic-date-table.vue'
 import MonthTable from './basic-month-table.vue'
 import YearTable from './basic-year-table.vue'
+import type { IDatePickerType } from '../date-picker.type.ts'
+import { IDatePickerTypeList } from '../date-picker.type'
 
-import {
```

Comment:
Do not mix type files with actual variables
separate them into two files would be better.
Also do: import variables above import types

Reply:
ok, i'll change it